codec_specs.md

```diff
@@ -58,6 +58,18 @@ A timestamp, expressed in [@!RFC3339] that notes when support for the `Codec Map
 
 A `Codec Mapping` MAY only be defined with a `Superseded By` value, if it has an expressed `Deprecation Date`. If used, the `Superseded By` value MUST store the `Codec ID` of another `Codec Mapping` that has superseded the `Codec Mapping`.
 
+## Recommendations for the Creation of New Codec Mappings
+
+Creators of new `Codec Mappings` to be used in the context of Matroska:
+
+- SHOULD assume that all `Codec Mappings` they create might become standardized, public, commonly deployed, or usable across multiple implementations.
+
+- SHOULD employ meaningful values for `Codec ID` and `Codec Name` that they have reason to believe are currently unused.
+
+- SHOULD NOT prefix their `Codec ID` with "X-" or similar constructs.
```

Comment:
`X-` is leftover from the similar recommendation from rfc6838. Yes `X_` would be more in Matroska style. Still we in addition to saying what not to do, we should recommend what to do. For example, enough info so that the poster of https://mailarchive.ietf.org/arch/msg/cellar/igOaI6HoCGx_K8nFceXMetoAoVQ could asssign a Codec ID.